composer.json

```diff
@@ -19,8 +19,8 @@
 	},
 	"scripts": {
 		"lint": "find . -name \\*.php -not -path './vendor/*' -not -path './build/*' -print0 | xargs -0 -n1 php -l",
-		"cs:check": "php-cs-fixer fix --dry-run --diff",
-		"cs:fix": "php-cs-fixer fix",
+		"cs:check": "vendor/bin/php-cs-fixer fix --dry-run --diff",
```

Comment:
composer pushes its vendor bin dir onto the $PATH, this shouldn't be necessary

Reply:
For phpunit at least that didn't work.